modules/presets/index.js

```diff
@@ -0,0 +1,5 @@
+export { Category } from './category.js';
+export { Collection } from './collection.js';
+export { Field } from './field.js';
+export { Preset } from './preset.js';
+export { presets } from './presets.js';
```

Comment:
Imho I feel we should have iD.presets modularized to 'presets/index.js'.
I have seen a general pattern in es6 modules, if you import a folder let say 'presets' the bundler automatically looks for 'presets/index.js'. So it would be better have all the functionality with iD.presets inside 'index.js'. And if you want you can have another file inside this folder called 'presets/presets' which has the functionality of iD.presets.presets or you can simply not have this file and do all the stuff in index.js. 

Let me know what are your views regarding this.